Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Playwright/PlaywrightBrowserElement.cs

```diff
@@ -1026,7 +1026,7 @@ public Task SetFileValueAsync(string value)
         /// <param name="playwrightLocator">The Playwright locator.</param>
         /// <param name="value">The file path to set.</param>
         /// <returns>A task representing the asynchronous operation.</returns>
-        private Task SetFileValueAsync(IPlaywrightLocator playwrightLocator, string value)
+        private Task SetFileValueAsync(IPlaywrightLocator playwrightLocator, string[] value)
```

Comment:
will this work against single file input?

Reply:
yes it's working, tested